[2024-06-21T04:41:45.166+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-21T04:41:45.191+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v3.task2 scheduled__2024-06-06T00:00:00+00:00 [queued]>
[2024-06-21T04:41:45.208+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v3.task2 scheduled__2024-06-06T00:00:00+00:00 [queued]>
[2024-06-21T04:41:45.211+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-21T04:41:45.225+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): task2> on 2024-06-06 00:00:00+00:00
[2024-06-21T04:41:45.231+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v3', 'task2', 'scheduled__2024-06-06T00:00:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp2ypy_o2r']
[2024-06-21T04:41:45.234+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=862) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-21T04:41:45.240+0000] {standard_task_runner.py:63} INFO - Started process 880 to run task
[2024-06-21T04:41:45.237+0000] {standard_task_runner.py:91} INFO - Job 84: Subtask task2
[2024-06-21T04:41:45.304+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v3.task2 scheduled__2024-06-06T00:00:00+00:00 [running]> on host fc356187513b
[2024-06-21T04:41:45.384+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v3' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2024-06-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-06T00:00:00+00:00'
[2024-06-21T04:41:45.389+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-21T04:41:45.404+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-21T04:41:45.405+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello world 2']
[2024-06-21T04:41:45.413+0000] {subprocess.py:86} INFO - Output:
[2024-06-21T04:41:45.418+0000] {subprocess.py:93} INFO - hello world 2
[2024-06-21T04:41:45.419+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-21T04:41:45.421+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-21T04:41:45.454+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v3, task_id=task2, run_id=scheduled__2024-06-06T00:00:00+00:00, execution_date=20240606T000000, start_date=20240621T044145, end_date=20240621T044145
[2024-06-21T04:41:45.485+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-21T04:41:45.499+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
